### PR TITLE
1499 display image description

### LINKED ISSUE #1499: Display image description in the View Details slide panel
Story https://github.com/magento/adobe-stock-integration/issues/724

### Preconditions (*)
1. Install magneto 2.4-develop with Adobe Stock Integration 2.0-develop
2. Enable Enhanced Media Gallery
3. Have an image uploaded to the media gallery
4. Have a description field filled in the `media_gallery_asset` table

### Steps to reproduce (*)
1. Login to admin panel
2. Open the media gallery (Content -> Media Gallery)
3. Select "View details" from the context menu for the image (opened clicking on three dots)

### Expected result (*)
The description should be visible in the image details panel

image

### AC
- section is displayed
- covered with MFTF tests

### PR DESCRIPTION
**Pull request by ghost**
### Description (*)
Currently the description of the images is not visible to the view details of the image i
add or retrieve image description from DB and display to View details page of image

### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1499: Display image description in the View Details slide panel
2. ...

### Manual testing scenarios (*)

1. Login to admin panel
2. Open the media gallery (Content -> Media Gallery)
3. Select "View details" from the context menu for the image (opened clicking on three dots)
4. You can see the Description section below the Details section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,5 @@ To connect with Magento team and the Community, join us on the [Magento Communit
 If you are interested in joining Slack, or a specific channel, use our [self signup](https://opensource.magento.com/slack) link.
 
 Adobe Stock Integration project slack channel: [#adobe-stock](https://magentocommeng.slack.com/messages/CJLSYEPFA)
+
+test
```
Comment (sivaschenko) [inline README.md]: Please remove the debug information

File: MediaGalleryUi/view/adminhtml/web/template/image/image-details.html [hunk 2]
```diff
@@ -27,6 +27,10 @@ <h3 data-bind="i18n: 'Details'"></h3>
                 </each>
             </div>
         </div>
+        <div class="description image-details-section">
```
Comment (sivaschenko) [inline MediaGalleryUi/view/adminhtml/web/template/image/image-details.html]: Please ensure the description is not displayed if it's empty

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the PR @gregbalonzo ! Please see my review comments

**Comment by sivaschenko** [comment]
@gregbalonzo can you please also sign the Adobe CLA - in the checks area below

File: README.md (hunk 1 above)
Comment (gabrieldagama) [inline README.md]:
```suggestion
```

File: MediaGalleryUi/view/adminhtml/web/template/image/image-details.html [hunk 3]
```diff
@@ -27,6 +27,10 @@ <h3 data-bind="i18n: 'Details'"></h3>
                 </each>
             </div>
         </div>
+        <div class="description image-details-section" if="image().description">
+            <h3 data-bind="i18n: 'Description'"></h3>
+                <div data-bind="text: image().description"></div>
```
Comment (gabrieldagama) [inline MediaGalleryUi/view/adminhtml/web/template/image/image-details.html]:
```suggestion
            <h3 data-bind="i18n: 'Description'"></h3>
            <div data-bind="text: image().description"></div>
```

File: MediaGalleryUi/view/adminhtml/web/template/image/image-details.html [hunk 4]
```diff
@@ -27,6 +27,10 @@ <h3 data-bind="i18n: 'Details'"></h3>
                 </each>
             </div>
         </div>
+        <div class="description image-details-section" if="image().description">
+            <h3 data-bind="i18n: 'Description'"></h3>
+            <div data-bind="text: image().description"></div>
```
Comment (gabrieldagama) [inline MediaGalleryUi/view/adminhtml/web/template/image/image-details.html]:
```suggestion
            <p data-bind="text: image().description"></p>
```

**Review by sivaschenko — APPROVED** [review]

**Comment by sivaschenko** [comment]
The MFTF test coverage will be provided in a separate issue, after the population of description field will be implemented.

@chalov-anton please add the description information to the `media_gallery_asset` table manually for testing purposes, also could you please create an MFTF coverage issue for this implementation (assuming description will be retrieved from the image during upload to the media gallery)

**Comment by sivaschenko** [comment]
@magento run all tests

**Comment by gabrieldagama** [comment]
@sivaschenko @chalov-anton I've created an issue for the MFTF tests on this one.

https://github.com/magento/adobe-stock-integration/issues/1529

**Comment by gabrieldagama** [comment]
@magento run Functional Tests CE,Functional Tests EE

**Review by gabrieldagama — APPROVED** [review]

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**